lib/grape-swagger/endpoint.rb

```diff
@@ -235,6 +235,11 @@ def parse_request_params(required)
       end
     end
 
+    def param_type_is_array?(param_type)
+      return false unless param_type
+      param_type == 'Array' || (param_type[0] == '[' && param_type[-1] == ']' && !param_type.include?(','))
```

Comment:
@dblock I could do a regex. The comma is because of a tricky situation. In Grape, you can specify type in 2 similar ways:

1. `type: Array[String]`, meaning an array of strings, and
1. `type: [String, Integer]`, meaning a multi-type param of either 'String' or 'Integer'

In the first case, once the options are within the realm of grape-swagger (at least from what I can tell), the `Array` prefix is stripped, and it just looks like `'[String]'`. The only way I could think to disambiguate `'[String]'` (array of type string) from `'[String, Integer]'` was to look for once vs. multiple items inside brackets, hence the ",". Open to other suggestions, but it's all I could think of.